internals/secrethub/service_init.go

```diff
@@ -73,8 +74,32 @@ func (cmd *ServiceInitCommand) Run() error {
 		return err
 	}
 
-	if cmd.permission != 0 {
-		_, err = client.AccessRules().Set(cmd.path.Value(), cmd.permission.String(), service.ServiceID)
+	permissionPath := cmd.path
+	var permission api.Permission
+	values := strings.SplitN(cmd.permission, ":", 2)
+	if len(values) == 1 {
```

Comment:
Is this the same functionality as used for AWS?

Reply:
The only difference between the implementations is that this command errors when the path arg is a directory and a subdirectory is given in the permission flag.

Reply:
I'll see tomorrow whether I can reduce duplication.

Reply:
It's now re-used between the commands